src/app/components/MediaLoader/configs/tvMedia.ts

```diff
@@ -0,0 +1,49 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: tvMediaBlock } = filterForBlockType(blocks, 'tvMedia');
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: tvMediaBlock.episodeTitle,
+    duration: tvMediaBlock?.versions?.[0]?.duration,
```

Comment:
I am wondering if we could extract `tvMediaBlock?.versions?.[0]` into its own variable and use it in all places below. 

I think we should do something like this (on line 11):
```
const video = tvMediaBlock?.versions?.[0] || {};
```

And then everywhere we reference `tvMediaBlock?.versions?.[0]`, we can replace it with `video`